e2e/live/end-to-end-test.sh

```diff
@@ -878,6 +878,38 @@ ${BIN_DIR}/kpt live apply e2e/live/testdata/continue-on-error > $OUTPUT_DIR/stat
 assertCMInventory "test-namespace" "1"
 assertPodExists "pod-a" "test-namespace"
 assertPodNotExists "pod-B" "test-namespace"
+
+# Test 21: RBAC error applying a resource
+echo "Testing RBAC error during apply"
+echo "kpt live apply e2e/live/testdata/rbac-error-step-1"
+echo "kpt live apply e2e/live/testdata/rbac-error-step-2"
+# Setup: create a service account and bind a Role to it so it has administrative
+# privileges on the "test" namespace, but no permissions on the default
+# namespace.
+kubectl apply -f e2e/live/testdata/rbac-error-step-1 > $OUTPUT_DIR/status
+assertContains "namespace/test created"
+assertContains "rolebinding.rbac.authorization.k8s.io/admin created"
+assertContains "serviceaccount/user created"
+wait 2
+
+# Setup: use the service account just created. It does not have permissions
+# on the default namespace, so it will give a permissions error on apply
+# for anything attempted to apply to the default namespace.
+kubectl config set-credentials user --token="$(kubectl get secrets -ojsonpath='{.data.token}' \
+  "$(kubectl get sa user -ojsonpath='{.secrets[0].name}')" \
+  | base64 -d)" > $OUTPUT_DIR/status
+kubectl config set-context kind-kind:user --cluster=kind-kind --user=user > $OUTPUT_DIR/status
```

Comment:
I think you can drop `> $OUTPUT_DIR/status` from these lines. Each of these commands will overwrite the contents of that output anyways so if we're not reading the contents the output shouldn't need to be saved.

Reply:
The main reason for the `> $OUTPUT_DIR/status`: it keeps the output from being polluted. We'd rather just have `.` or `E` than the entire command output, which is what would happen without the redirection. This could probably be changed with a `verbose` flag, but I haven't created it yet.